tests/test_timeseriesflattener/test_resolve_multiple.py

```diff
@@ -154,3 +154,21 @@ def test_resolve_multiple_count():
         lookahead_days=2,
         expected_flattened_values=[2],
     )
+
+
+def test_resolve_multiple_slope():
+    prediction_times_str = """dw_ek_borger,timestamp,
+                            1,2021-12-31 00:00:00
+                            """
+    event_times_str = """dw_ek_borger,timestamp,val,
+                        1,2022-01-01 00:00:01, 1
+                        1,2022-01-02 00:00:02, 2
+                        """
```

Comment:
Suggest making the time 00:00:00 to make it easier to determine that it's slope per day :-)

Suggest adding tests for:
1. Negative slopes
2. If the event-times are not ordered correctly in the input